### ISSUE TITLE
Accessibility: Add playwright report

### ISSUE THREAD

**Issue by jredingcsv** [CognizantOpenSource/maqs-java#108]
We currently have support for Selenium accessibility Axe reporting, now we should be able to do that for Playwright
